syntax = "proto3";

option optimize_for = LITE_RUNTIME;

package GameServerPB;

enum eGameMsgType
{
    eUnknown       = 0;
    eLogin         = 1;  // Initial login request
    eLoginAck      = 2;  // Initial login acknowledge
} // When adding new enum updatge GameSerializer::msgType()

enum eSucces
{
    eNO               = 0;
    eYES              = 1;
}

message login
{
    uint32 wire_ver   = 1;      // Internal wire format
    string guid       = 2;      // Game Type for routing
    string user       = 3;
    string pasword    = 4;
}

message login_ack
{
    eSucces success   = 1;
    string  message   = 2;
}


